server/src/main/java/org/opensearch/node/remotestore/RemoteStorePinnedTimestampService.java

```diff
@@ -262,6 +276,10 @@ public static Tuple<Long, Set<Long>> getPinnedTimestamps() {
         return pinnedTimestampsSet;
     }
 
+    public static Map<String, List<Long>> getPinnedEntities() {
```

Comment:
Can we verify the contents of this map as part of tests in `RemoteStorePinnedTimestampsIT`? In this test class, we assert contents of `getPinnedTimestamps` at multiple places. At the same time, we can assert contents of output of this method as well.